<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="com.tim.one.advice.TestAfterThrowingAdvice" tests="1" skipped="0" failures="0" errors="0" timestamp="2015-05-20T16:40:13" hostname="josdem-Inspiron-5420" time="0.029">
  <properties/>
  <testcase name="shouldSendCryptoMessage" classname="com.tim.one.advice.TestAfterThrowingAdvice" time="0.029"/>
  <system-out><![CDATA[678  [Test worker] INFO  com.tim.one.advice.AfterThrowingAdvice  - Wrapping exception message
]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
